Content.Shared.Database/LogType.cs

```diff
@@ -479,6 +479,7 @@ public enum LogType
     /// </summary>
     Instrument = 103,
 
+    RMCAdminCommandLogging = 99999,
```

Comment:
For this you should make it RMCMarineAnnounce + the next number, so + 39 in this branch